packages/components/src/components/button/button.jsx

```diff
@@ -64,7 +66,7 @@ const Button = ({
             type={is_submit_success ? 'button' : type || 'submit'}
             {...props}
         >
-            {icon && <div className='dc-btn__icon'>{icon}</div>}
+            {icon && <div className={is_circle ? 'dc-btn__icon--circle' : 'dc-btn__icon'}>{icon}</div>}
```

Comment:
```
<div className={classNames('dc-btn__icon', { 'dc-btn__icon--circle' : is_circle })}>
```
IMO we should still have `dc-btn__icon` since `is_circle` is just a modifier